news/_posts/2019-09-11-prestashop-in-2019-and-beyond-part-2-pain-points.md

```diff
@@ -32,29 +32,29 @@ Yes, I have met people who download it and use it as-is, but I think they belong
 
 This means that we, the people who work on PrestaShop's Core code, have to keep three kinds of "users" into account:
 
-1. The shoppers that buy things on PrestaShop shops,
-2. ... the merchants that run PrestaShop shops (and ultimately, pay for),
-3. ... and the **developers** that work using PrestaShop to build sites, or develop modules and themes.
+1. The shoppers who buy things on PrestaShop shops,
+2. ... the merchants who run PrestaShop shops (and ultimately, pay for),
+3. ... and the **developers** who use PrestaShop to build sites, or develop modules and themes.
 
 This third "user", is very, _very_ important to us. [I mean it like Steve Ballmer](https://www.youtube.com/watch?v=1VgVJpVx9bc).
 
 It's not just because we _love_ developers (incidentally, we do!). It's because PrestaShop cannot survive without them. As such, we want to encourage them to develop for our platform. 
 
-As platform developers, we face a big challenge that most developers (or at least, web developers) usually don't need to care about: **backwards compatibility**.
+As platform developers, we face a big challenge that most developers (or at least, web developers) usually don't need to care about: **backward compatibility**.
```

Comment:
I've been writing it wrong all along!

Reply:
I was having doubts about it, but even [Wikipedia's own page on the topic](https://en.wikipedia.org/wiki/Backward_compatibility) uses "backward compat" more than "backwards compat" -- so it seems there's no definitive, but it's clearly leaning on the "backward" side of things. 
I guess this is a case for a native speaker :)